src/js/utils/filter-source.js

```diff
@@ -56,10 +56,12 @@ const filterSource = function(src) {
  *        src Object with known type
  */
 function fixSource(src) {
-  const mimetype = getMimetype(src.src);
+  if (!src.type) {
+    const mimetype = getMimetype(src.src);
```

Comment:
Even sources with a `type` were effected because we always got the mimetype for every source, even if we did not use it. This change makes it so we only get a type when we don't have one.